charts/camunda-platform-8.8/templates/orchestration/data-migration-configmap.yaml

```
@@ -0,0 +1,64 @@
+{{- if .Values.orchestration.migration.data.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "orchestration.fullname" . }}-process-migration-configuration
+data:
+  application.yaml: |
+    {{- if .Values.global.elasticsearch.enabled }}
+    camunda:
+      database:
+        connect:
+          cluster-name: elasticsearch
+          cluster-type: elasticsearch
+        url: {{ include "camundaPlatform.elasticsearchURL" . | quote }}
+        {{- if and .Values.global.elasticsearch.auth.username .Values.global.elasticsearch.auth.password }}
+        username: {{ .Values.global.elasticsearch.auth.username }}
+        password: ${VALUES_DATABASE_PASSWORD:}
+        {{- end }}
+    {{- else }}
+    camunda:
+      database:
+        connect:
+          cluster-name: opensearch
+          cluster-type: opensearch
+        url: {{ include "camundaPlatform.opensearchURL" . | quote }}
+        {{- if and .Values.global.opensearch.auth.username .Values.global.opensearch.auth.password }}
+        username: {{ .Values.global.opensearch.auth.username }}
+        password: ${VALUES_DATABASE_PASSWORD:}
+        {{- end }}
+    {{- end }}
+      migration:
+        process:
+          batch-size: {{ .Values.orchestration.migration.data.process.batchSize }}
+          import-finished-timeout: {{ .Values.orchestration.migration.data.process.importFinishedTimeout }}
```

```suggestion
          importer-finished-timeout: {{ .Values.orchestration.migration.data.process.importFinishedTimeout }}
```

Same for all other occurences

---

Thanks. I made the change.